src/trip.js

```diff
@@ -10,6 +10,7 @@ class Trip {
     this.duration = tripData.duration
     this.status = tripData.status
     this.suggestedActivities = tripData.suggestedActivities
+    this.destinationData = destinationData
```

Comment:
I like how you have `destinationData` as part of your constructor.  Makes it really easy to work with moving forward.